dashboards/wildfly/README.md

```diff
@@ -0,0 +1,13 @@
+### Dashboards for WildFly
+
+#### Notes
+
+- This dashboard is based on Google's [Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent).
+- The full list of metrics supported by the Ops Agent can be found [here](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/wildfly#monitored-metrics).
+
+
+|WildFly GCE Overview|
+|:------------------|
+|Filename: [wildfly-gce-overview.json](wildfly-gce-overview.json)|
+|This dashboard has 12 charts for viewing wildfly when monitored by 
+[Google's Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/wildfly#monitored-metrics), including `Active Sessions`, `Expired Sessions`, `Rejected Sessions`, `Requests`, `Request Errors`, `Request Time`, `Network IO`, `JDBC Connections`, `JDBC Request Waits`, `JDBC Transactions`, `JDBC Rollbacks`, `CPU % Top 5 VMs`, `Memory % Top 5 VMs` and `Hosts by Region`. There is also a `WildFly Monitoring Link` card with links to docs and WildFly log in Cloud Logging.
```

Comment:
This newline creates an extra line in the markdown table, can you remove it?

![image](https://user-images.githubusercontent.com/1757661/155557863-ca75c0ce-0607-42cb-8f60-d9a37924701b.png)

Reply:
Thank you!